client/src/components/CardLeftImage.js

```diff
@@ -15,15 +15,7 @@ const CardLeftImage = ({
 }) => (
   <a className="link-unstyled" href={link_href}>
     <div className="left-img-card">
-      {img_src && (
-        <div className="left-img-card__left">
-          <img
-            aria-hidden="true"
-            src={img_src}
-            className="left-img-card__img"
-          />
-        </div>
-      )}
+      {svg && <div className="left-img-card__left">{svg}</div>}
```

Comment:
Used to be aria-hidden, but isn't any more.

Hmm, maybe the icons should default to aria-hidden if they aren't supplied a `title` prop.

Reply:
This happens in a number of places in this branch, not going to point them all out, changing the defaults will cover them all.